powerline/segments/i3wm.py

```diff
@@ -1,9 +1,9 @@
 # vim:fileencoding=utf-8:noet
 from __future__ import (unicode_literals, division, absolute_import, print_function)
 
-import i3
-
+import i3ipc
 
+conn = i3ipc.Connection()
```

Comment:
Why? You need to just `global conn; if conn is None: conn = i3ipc.Connection()` and `conn = None` here. I did not say that `conn` must not be a global, I said that connection must not be created at import.